src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.js

```diff
@@ -0,0 +1,81 @@
+import {LightningElement, api, wire} from 'lwc';
+import {getRecord, getFieldValue} from 'lightning/uiRecordApi';
+import {NavigationMixin} from 'lightning/navigation';
+
+import NAME_FIELD from '@salesforce/schema/DataImportBatch__c.Name';
+import BATCH_DRY_RUN_LABEL from '@salesforce/label/c.bgeBatchDryRun';
+import PROCESS_BATCH_LABEL from '@salesforce/label/c.bgeProcessBatch';
```

Comment:
We definitely _could_ add these labels into the geLabelService component and add it in, but we are currently only using these three and three others on the geFormRenderer component on the form (single or batch mode) so I'm a little hesitant to import that component with its 100+ other labels that are not used in this component hierarchy.  But, let me know if you think the benefits of that alignment are worthwhile and I can refactor.

Reply:
I don't really know what the cost of that import is, which makes it hard to make a smart statement - since they may be responsibly cached by the platform and we are probably referencing the majority of them at SOME point or another in the application, it's possible it doesn't actually cost much to have them all grouped through the geLabelService - but really not sure. I'm not inclined to be prescriptive based on what I know now, although we may want to do some work to understand if there are drawbacks to having a couple hundred labels that we always load in every component.